Lib/socketserver.py

```diff
@@ -438,9 +438,10 @@ class TCPServer(BaseServer):
 
     socket_type = socket.SOCK_STREAM
 
-    request_queue_size = 5
+    request_queue_size = 0
 
-    allow_reuse_address = False
+    allow_reuse_address = \
+        os.name not in ('nt', 'cygwin') and hasattr(socket, 'SO_REUSEADDR')
```

Comment:
Why the OS check here? Is `SO_REUSEADDR` available but not appropriate for use on Windows? If so, I suggest adding a comment.

Also, why use `os.name` rather than `sys.platform`?